qtpyvcp/plugins/notifications.py

```diff
@@ -157,20 +158,22 @@ def timerEvent(self, event):
 
     def initialise(self):
 
+        if self.persistent:
+            self.messages = self.data_manager.getData('messages', [])
+
+        self.startTimer(200)
+
+    def postGuiInitialise(self):
+        mainwindow = qtpyvcp.WINDOWS.get('mainwindow')
         if self.enabled:
             if self.mode == "native":
-                self.notification_dispatcher = NativeNotification()
+                self.notification_dispatcher = NativeNotification(parent=mainwindow)
```

Comment:
this here is definitely cleaner. But we could also just set the parent in the timer or in captureMessage on-demand.